src/pages/operators/{contentfulOperatorAnalysis.operator__name}.tsx

```diff
@@ -398,6 +398,11 @@ const globalOverrideStyles =
       a {
         color: ${accentColor};
       }
+      
+      .heading-block {
+        background: linear-gradient(90deg, ${transparentize(0.9, accentColor)}, ${transparentize(1, theme.palette.white)}), ${transparentize(0.67, theme.palette.midtoneBrighter)};
+        backdrop-filter: blur(8px);
```

Comment:
We could use `blur(${theme.spacing(1)};` here, but I dunno if that's preferable (would we want the blur to scale up with changes to spacing unit? what do we do elsewhere in the code for other `backdrop-filter`s?)

Reply:
immediately under, it's `theme.spacing(1)`, so let's use that here too for the sake of consistency

Reply:
never mind. it's already blurred kekw